packages/manager/apps/web/client/app/hosting/multisite/translations/Messages_fr_FR.json

```
@@ -10,11 +10,10 @@
   "hosting_multisite_statistics_period_year": "Année",
   "hosting_multisite_dialog_title": "Vérification des serveurs DNS",
   "hosting_multisite_dialog_close": "Fermer",
-  "hosting_multisite_dialog_modify_a_record": "Modifiez le champ de type {{recordType}} avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} géré sur les serveurs DNS {{dns}}",
-  "hosting_multisite_dialog_modify_a_record_same_nic": "Modifier le champ de type {{recordType}}",
-  "hosting_multisite_dialog_modify_a_record_external_nic": "Modifiez le champ de type {{recordType}} avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} géré sur les serveurs DNS {{dns}} (la modification doit être effectuée par l'administrateur de la zone DNS: {{nic}})",
-  "hosting_multisite_dialog_modify_a_record_error": "Ajoutez un champ de type {{recordType}} avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} géré sur les serveurs DNS {{dns}}",
-  "hosting_multisite_dialog_modify_a_record_same_nic_error": "Modifier le champ de type {{recordType}}",
+  "hosting_multisite_dialog_edit_record_description": "Modifiez le champ de type {{ recordType }} avec l'adresse IP {{ ip }} dans la zone DNS du domaine {{ domain }} géré sur les serveurs DNS {{ zone }}.",
+  "hosting_multisite_dialog_edit_record_link": "Modifier le champ de type {{ recordType }}.",
```

Different tenses in the same page, normal ?
```suggestion
  "hosting_multisite_dialog_edit_record_link": "Modifiez le champ de type {{ recordType }}.",
```

---

It's link that not in the text description it's "as button go to".